examples/src/test/scala/za/co/absa/atum/HdfsInfoIntegrationSuite.scala

```diff
@@ -44,17 +40,18 @@ class HdfsInfoIntegrationSuite extends AnyFlatSpec with SparkTestBase with Match
   }
 
   private val inputCsv = "data/input/wikidata.csv"
+
   private def readSparkInputCsv(inputCsvPath: String): DataFrame = spark.read
     .option("header", "true")
     .option("inferSchema", "true")
     .csv(inputCsvPath)
 
-  private def writeSparkData(df: DataFrame, outputPath: String): Unit = {
+  private def writeSparkData(df: DataFrame, outputPath: String, implicitPath: Option[String] = None): Unit = {
     df.write.mode(SaveMode.Overwrite)
       .parquet(outputPath)
 
     eventually(timeout(scaled(10.seconds)), interval(scaled(500.millis))) {
-      if (!Files.exists(Paths.get(outputPath)))
+      if (!Files.exists(Paths.get(outputPath)) || implicitPath.exists(x => !Files.exists(Paths.get(x))))
```

Comment:
Why this change? I don't see it used anywhere. Is there a future use case or perhaps there was one and it got removed in the meantime?

Reply:
Deleted on revert. Will get it back there.